src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.model;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
```

Comment:
Lets make it of type `Color` since we already have an enum for it, its is preferred to use it instead of String